Makefile

```diff
@@ -28,6 +30,14 @@ backend: $(FRONTEND_TARGET)
 run: backend
 	dune exec -- ftw --db=tests/test.db
 
+frontend_dev: backend
+	@(dune exec -- ftw --db=tests/test.db > ftw.log 2>&1 & echo $$! > ftw.pid; \
```

Comment:
maybe you would write a shell script to do this ? that would be much easier to write/read/understand and modify in the future given the complexity of the invocations